test/unit/sample.test.ts

```diff
@@ -0,0 +1,31 @@
+import { fixture, html } from '@vaadin/vaadin-component-dev-dependencies/testing.js';
+import { VaadinComponent } from '../../src/vaadin-component';
+
+const { expect } = chai;
+
+describe('sample test', () => {
+  let element: VaadinComponent;
+  let tagName: string;
+
+  beforeEach(async () => {
+    element = await fixture(
+      html`
+        <vaadin-component></vaadin-component>
+      `
+    );
+    tagName = element.tagName.toLowerCase();
+  });
+
+  it('should be defined in custom element registry', () => {
+    expect(customElements.get(tagName)).to.be.ok;
+    expect(element instanceof VaadinComponent).to.be.ok;
+  });
+
+  it('should have a valid static "is" getter', () => {
+    expect(customElements.get(tagName).is).to.equal(tagName);
+  });
+
+  it('should have a valid version number', () => {
+    expect(customElements.get(tagName).version).to.match(/^(\d+\.)?(\d+\.)?(\d+)(-(alpha|beta)\d+)?$/);
+  });
```

Comment:
These should probably be wrapped in a `describe('custom element definition', () => {` block same as in [accordion PR](https://github.com/vaadin/vaadin-accordion/pull/30/files#diff-db5b90a4a9e659e4115ef9fe65ad6e86R37-R38) and [details](https://github.com/vaadin/vaadin-details/blob/next/test/unit/details.test.ts#L23-L24) to make it easier to stay consistent when creating new components based on this.

Reply:
Done.